.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
```

Comment:
I think a small Readme (or a small section in the main Readme) would be nice on how to actually utilize this file.

Reply:
Thats a point, I will put in a reference to the official manual

Reply:
fixed